ctapipe/image/hillas.py

```diff
@@ -197,9 +194,24 @@ def hillas_parameters(geom, image):
             np.sum(((((b * A) + (a * B) + (-c * C))) ** 2.0) * image)
         ) / (2 * width)
 
+    if unit.is_equivalent(u.m):
+        return CameraHillasParametersContainer(
+            x=u.Quantity(cog_x, unit),
+            y=u.Quantity(cog_y, unit),
+            r=u.Quantity(cog_r, unit),
+            phi=Angle(cog_phi, unit=u.rad),
+            intensity=size,
+            length=u.Quantity(length, unit),
+            length_uncertainty=u.Quantity(length_uncertainty, unit),
+            width=u.Quantity(width, unit),
+            width_uncertainty=u.Quantity(width_uncertainty, unit),
+            psi=Angle(psi, unit=u.rad),
+            skewness=skewness_long,
+            kurtosis=kurtosis_long,
+        )
     return HillasParametersContainer(
-        x=u.Quantity(cog_x, unit),
-        y=u.Quantity(cog_y, unit),
+        fov_lon=u.Quantity(cog_x, unit),
```

Comment:
Please double check that this association (fov_lon with x) is correct. I remember it the other way around but I am not sure.

Reply:
I think you are correct in general. Will check tomorrow if the x/y are consistent here

Reply:
Sorry was mistaken. Or confused. 

This is correct. If a camera geometry is in telescope frame, `pix_x` refers to `TelescopeFrame.fov_lon`. 

I was confused by the fact that in the `CameraFrame`, pix_x is refers to (obviously) `CameraFrame.x` which is aligned with `fov_lat`.